src/library-authoring/units/LibraryUnitBlocks.tsx

```diff
@@ -23,6 +24,15 @@ import { LibraryBlock } from '../LibraryBlock';
 import { useLibraryRoutes } from '../routes';
 import messages from './messages';
 
+/** Components that need large min height in preview */
+const LARGE_COMPONENTS = [
+  COMPONENT_TYPES.advanced,
+  COMPONENT_TYPES.dragAndDrop,
+  COMPONENT_TYPES.discussion,
+  'lti',
+  'lti_consumer',
```

Comment:
@ChrisChV We just need to add block types that need larger min height, let me know if I missed any.